recipes/business-duration/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "business-duration" %}
+{% set version = "0.66" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/e5/1d/12b3bb612a19d20ca2d9fe7ebfc80ab6b9e1d5d575c6a3dcd121c7cbfed4/business_duration-0.66.tar.gz"
+  sha256: "2fbecabe8059a259623c357b1c4f4274408498bfca52647e39ec10326182d74b"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - business_duration
+
+about:
+  home: "https://github.com/gnaneshwar441/Business_Duration"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "PKG-INFO"
```

Comment:
This doesn't sound like a typical license file.

Reply:
It's not, it's only the license metadata from packaging, no actual text with language of the license.